plugins/base/src/main/kotlin/translators/documentables/DefaultDocumentableFilteringStrategies.kt

```diff
@@ -0,0 +1,9 @@
+package org.jetbrains.dokka.base.translators.documentables
+
+import org.jetbrains.dokka.model.DFunction
+import org.jetbrains.dokka.model.PrimaryConstructorExtra
+
+object DefaultDocumentableFilteringStrategies : DocumentableFilteringStrategies {
+    override fun shouldConstructorBeInPage(constructor: DFunction): Boolean =
+        constructor.extra[PrimaryConstructorExtra] == null || constructor.documentation.isNotEmpty()
```

Comment:
I'd leave a comment explaining the check. Not too obvious for someone encountering this for the first time
```suggestion
        // primary kotlin constructors are rendered as part of the class signature
        // and are not included in any separate constructor blocks
        constructor.extra[PrimaryConstructorExtra] == null || constructor.documentation.isNotEmpty()
```